src/sdk/pynni/nni/nas/pytorch/modules.py

```diff
@@ -0,0 +1,9 @@
+
+from torch import nn as nn
+
+
+class RankedModule(nn.Module):
```

Comment:
What is RankedModule?

Reply:
like cell, it marks difference rank/level of group. for example, in darts, cell is rank=1, and ops can be treat like rank=2. It can support macro/micro pattern better.

Reply:
Where is this `rank/level` used?

Reply:
I thought to update on existing network for pdarts implementation, so added it. But finally, to get exact implementation, the pdarts recreate network each time. So it's not used in this time, but I believe it's a good mark for future implementation.

Reply:
Looks "over-engineering" to me. I'm fine with it. Just make everything clear in the docstring. :)